tests/osaka/eip7692_eof_v1/eip7069_extcall/test_gas.py

```diff
@@ -140,3 +140,36 @@ def test_ext_calls_gas(
         cold_gas=cold_gas + cost_memory_bytes(mem_expansion_bytes, 0),
         warm_gas=warm_gas + cost_memory_bytes(mem_expansion_bytes, 0),
     )
+
+
+@pytest.mark.parametrize("value", [0, 1])
+def test_extdelegatecall_never_charges_transfer_gas(
+    state_test: StateTestFiller,
+    pre: Alloc,
+    state_env: Environment,
+    value: int,
+):
+    """
+    Test that EXTDELEGATECALL call doesn't charge for value transfer, even if the outer call
+    transfered value.
+    """
+    noop_callee_address = pre.deploy_contract(Container.Code(Op.STOP))
+
+    extdelegatecall_contract_address = pre.deploy_contract(
+        Container.Code(Op.EXTDELEGATECALL(address=noop_callee_address) + Op.STOP)
```

Comment:
Should we test `Op.EXTSTATICCALL` too in the same way?

Reply:
We could, I don't think there would be any obstacles, except the "intent" of the test stems from EXTDELEGATECALL preserving value of the outer call. I can add EXTSTATICCALL and make a comment that it is just an addition for good measure. And actually, we can throw in EXTCALL without value arg set too, all 3 should have same effect (no 9000 charged). I think this makes sense, with an appropriate comment, one sec...